content/scalers/huawei-cloudeye.md

```diff
@@ -0,0 +1,120 @@
++++
+fragment = "content"
+weight = 100
+title = "Huawei Cloudeye"
+background = "light"
++++
+
+Scale applications based on a Huawei Cloudeye.
+
+**Availability:** v1.0+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `huawei-cloudeye` trigger that scales based on a Huawei Cloudeye.
+
+```yaml
+triggers:
+  - type: huawei-cloudeye
+    metadata:
+      namespace: SYS.ELB
+      dimensionName: lbaas_instance_id
+      dimensionValue: 5e052238-0346-xxb0-86ea-92d9f33e29d2
+      metricName: mb_l7_qps
+      targetMetricValue: "100"
+      minMetricValue: "1"
+```
+
```

Comment:
Would it be possible to provide some brief description on what field means what? They are kind of straight forward but doesn't hurt.

Not mandatory but nice to have.

Reply:
thanks. done it.

Reply:
Nitpick - Can you do it in the text rather than YAMLs similar to other scalers please?

https://keda.sh/scalers/azure-service-bus/

Reply:
hi @tomkerkhove , i add parameter list.

Reply:
Thanks man!

Reply:
Awesome, thanks!